src/templates/toolkit/bet/static.html.tt

```diff
@@ -154,7 +154,7 @@
                         <div class="row-inner big-col">
                            <div class="col-inner">
                               <select id="currency" class="small_width_input"></select>
-                              <input id="amount" type="number" step="any" value="10" class="medium_width_input" />
+                              <input id="amount" type="text" step="any" maxlength="10" value="10" class="medium_width_input" />
```

Comment:
Setting maxlength to restrict input. Maxlength of valid payout is 8 for `99999.99` rounded it off to nearest tens place.